src/__swaps__/screens/Swap/resources/search/searchV2.ts

```diff
@@ -77,29 +72,6 @@ export const useTokenSearchStore = createQueryStore<VerifiedTokenData, TokenSear
   { persistThrottleMs: time.seconds(8), storageKey: 'verifiedTokenSearch' }
 );
 
-export const useUnverifiedTokenSearchStore = createQueryStore<SearchAsset[], TokenSearchParams<TokenLists.HighLiquidity>>(
```

Comment:
this is the removal of the "extra" call for assets on the unverified list - these results will now show up as part of the main call to `tokenSearchQueryFunction`